pkg/api/api.go

```diff
@@ -252,9 +252,9 @@ func (hs *HTTPServer) registerRoutes() {
 
 		apiRoute.Get("/datasources/id/:name", Wrap(GetDataSourceIdByName), reqSignedIn)
 
-		apiRoute.Get("/plugins", Wrap(hs.GetPluginList))
-		apiRoute.Get("/plugins/:pluginId/settings", Wrap(GetPluginSettingByID))
-		apiRoute.Get("/plugins/:pluginId/markdown/:name", Wrap(GetPluginMarkdown))
+		apiRoute.Get("/plugins", reqOrgAdmin, Wrap(hs.GetPluginList))
+		apiRoute.Get("/plugins/:pluginId/settings", reqOrgAdmin, Wrap(GetPluginSettingByID))
+		apiRoute.Get("/plugins/:pluginId/markdown/:name", reqOrgAdmin, Wrap(GetPluginMarkdown))
```

Comment:
If you move this into the plugins route block below they will get the same route protection.

Think the markdown is used from queries view, and panel visualization settings also has a help button that can request plugin markdown so think that should not require org admin

Reply:
Good point fixed